scripts/dss_v2_apdapter.py

```diff
@@ -140,13 +144,10 @@ def _parse_staging_area(self) -> Tuple[str, str]:
         Path value will not have a prefix '/' and will have a postfix '/' if not empty.
         """
         split_url = parse.urlsplit(self.args.staging_area)
-        if split_url.scheme != 'gs' or not split_url.netloc:
-            raise ValueError('Staging area must be in gs://<bucket>[/<path>] format')
-        elif split_url.path.endswith('/'):
-            raise ValueError('Staging area URL must not end with a "/"')
-        else:
-            path = f"{split_url.path.lstrip('/')}/" if split_url.path else ''
-            return split_url.netloc, path
+        require(split_url.scheme == 'gs' and split_url.netloc, 'Staging area must be in gs://<bucket>[/<path>] format')
```

Comment:
```suggestion
        require(split_url.scheme == 'gs' and split_url.netloc, 
                'Staging area must be in gs://<bucket>[/<path>] format')
```

Here and everywhere the lines get longish